charts/karavi-observability/templates/karavi-observability-configmap.yaml

```diff
@@ -54,6 +54,9 @@ data:
     POWERSTORE_VOLUME_IO_POLL_FREQUENCY: "{{ .Values.karaviMetricsPowerstore.volumePollFrequencySeconds }}"
     POWERSTORE_VOLUME_METRICS_ENABLED: "{{ .Values.karaviMetricsPowerstore.volumeMetricsEnabled }}"
     POWERSTORE_MAX_CONCURRENT_QUERIES: "{{ .Values.karaviMetricsPowerstore.concurrentPowerstoreQueries }}"
+    POWERSTORE_SPACE_POLL_FREQUENCY: "{{ .Values.karaviMetricsPowerstore.spacePollFrequencySeconds }}"
```

Comment:
Should these also have an enable flag similar to 'volumeMetricsEnabled'?

Reply:
Using the same volume metrics flag, reason I do not see if volume metrics is disable they will enable space or vice versa.